src/components/Pinpadkey/PinpadKey.tsx

```diff
@@ -0,0 +1,43 @@
+import styles from "./Pinpadkey.module.css";
+import Backspace from "@mui/icons-material/Backspace";
+import Check from "@mui/icons-material/Check";
+import { PINPAD_MUI_KEYS } from "../../utils/constants";
+
+interface PinpadKeyProps {
+	pinkey: string;
+	onHandleKeyPress: (key: string) => void;
+}
+
+export const PinpadKey: React.FC<PinpadKeyProps> = ({
+	pinkey,
+	onHandleKeyPress,
+}) => {
+	if (pinkey === PINPAD_MUI_KEYS.BACKSPACE) {
+		return (
+			<div
```

Comment:
Could these be `button`s? https://www.yanandcoffee.com/2020/08/10/button-vs-div/
I'm not certain, but I suspect that MR headsets might be able to e.g. snap to buttons for input, while divs wouldn't be as precise.